tiered-storage/jcloud/src/main/java/org/apache/bookkeeper/mledger/offload/jcloud/impl/BlobStoreBackedReadHandleImpl.java

```diff
@@ -119,7 +122,9 @@ public CompletableFuture<LedgerEntries> readAsync(long firstEntry, long lastEntr
                     getId(), firstEntry, lastEntry, (1 + lastEntry - firstEntry));
         }
         CompletableFuture<LedgerEntries> promise = new CompletableFuture<>();
+        touch();
```

Comment:
@poorbarcode It's addressed here.

Reply:
Answered here: https://github.com/apache/pulsar/pull/19783#discussion_r2103670948